Makefile

```
@@ -1,5 +1,13 @@
 all: bin/dver bin/dreg
 
+TAG=core-dreg:latest
```

I think it's better to tag it with `local`, as testing the latest locally built image and the latest on dockerhub maybe confusing.

```suggestion
TAG=core-dreg:local
```